mapsme_generator/data/car.lua

```diff
@@ -0,0 +1,437 @@
+-- Begin of globals
```

Comment:
please remove all binary files which are not specifically changed to work with `osm-export-tool2`, and which are part of basic maps.me installation